spec/features/join_game_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.feature 'join a game' do
+  let(:user1) { FactoryGirl.create(:user) }
+  let(:user2) { FactoryGirl.create(:user) }
+  let(:game) { FactoryGirl.create(:game, white_player_id: user1.id) }
+
+  context 'when available' do
+    it 'allows user to join' do
+      login_as(user2, scope: :user)
+      visit game_path(game)
+      expect(page).to have_text 'Only one more player needed!'
+
+      find_link('Join Game', visible: :all).visible?
```

Comment:
Picked it up in the capybara readme under [Finding](https://github.com/teamcapybara/capybara#finding). Scrolling up... looks like capybara [locates visible elements by default](https://github.com/teamcapybara/capybara#the-dsl), so seems redundant.